sensorhub-driver-android/src/main/java/org/sensorhub/impl/sensor/android/video/AndroidCameraOutputH265.java

```diff
@@ -39,16 +42,25 @@ public AndroidCameraOutputH265(AndroidSensorsDriver parentModule, int cameraId,
         super(parentModule,cameraId,previewTexture,"camera" + cameraId + "_H265");
     }
 
+    @Override
+    protected void initOutputStructure() {
+        // create SWE Common data structure and encoding
+        VideoCamHelper fac = new VideoCamHelper();
+        // use the one of H264
+        DataStream videoStream = fac.newVideoOutputH264(getName(), imgWidth, imgHeight);
```

Comment:
Need to update to set H265 as codec in output encoding description

Reply:
Done using generic descriptor